WordPress/src/main/java/org/wordpress/android/ui/plugins/PluginDetailActivity.java

```diff
@@ -124,6 +141,16 @@ public void onCheckedChanged(CompoundButton compoundButton, boolean b) {
             }
         });
 
+        mUpdateVersionTextView.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                if (isUpdateAvailable()) {
```

Comment:
We should use `NetworkUtils.checkConnection()` here to alert the user when they try to update without a connection.

Reply:
Added in fca81a8261f262b3735df3745560f94bb7a43d51